easybuild/easyconfigs/e/elbencho/elbencho-2.2-5-GCCcore-11.3.0.eb

```
@@ -0,0 +1,39 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Maicon Faria <maicon.faria@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+
+name = 'elbencho'
+version = '2.2-5'
+
+homepage = 'https://github.com/breuner/elbencho/r'
+description = "elbencho was inspired by traditional storage benchmark tools like fio, mdtest and ior, but was written"\
+              "from scratch to replace them with a modern and easy to use unified tool for file systems, object stores"\
+              "block devices."
```

```suggestion
              block devices."""
```
We actually handle long lines in a different way, which I've indicated in the suggestions.